test/integration/http/github-repo-enabled-payload-test.js

```diff
@@ -0,0 +1,85 @@
+const path = require('path')
```

Comment:
See my comment in `test/autotests/http/github-repo-enabled/enabled/test.js` before reading this comment.

If we go with my suggestions in the above mentioned comment, this file could become `test/integration/http/repository-enabled-test.js`. (if we think we'll add non-payload http/ tests in the future, we may need to throw this in `autotests/` or a new `payload/` directory)

Of course, this will mean this file will eventually need to be made more generic and/or call out to provider specific setup/teardown utils if necessary since it is currently very github specific. However, this more generic/single file for a route seems like the right approach since we'd be enumerating the context/test cases within `autotests/<route>/*`. Thoughts?